specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2280 @@
+          ]
+        },
+        "bandwidthMHz": {
+          "type": "number",
+          "description": "Bandwidth in MHz.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "direction": {
+          "type": "string",
+          "description": "Direction (uplink or downlink).",
+          "enum": [
+            "uplink",
+            "downlink"
+          ],
+          "x-ms-enum": {
+            "name": "direction",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "polarization": {
+          "type": "string",
+          "description": "Polarization. e.g. (RHCP, LHCP).",
+          "enum": [
+            "RHCP",
+            "LHCP",
+            "linearVertical",
+            "linearHorizontal"
+          ],
+          "x-ms-enum": {
+            "name": "polarization",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "authorizations": {
+          "type": "array",
+          "description": "Authorized Ground Stations",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/AuthorizedGroundstation"
+          },
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "List of authorized spacecraft links per ground station and the expiration date of the authorization."
+    },
+    "Spacecraft": {
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SpacecraftsProperties",
+          "description": "Spacecraft Properties",
+          "allOf": [
+            {
+              "$ref": "#/definitions/SpacecraftsProperties"
+            }
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Customer creates a spacecraft resource to schedule a contact."
+    },
+    "ResourceReference": {
+      "type": "object",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "Resource Reference."
+    },
+    "EndPoint": {
+      "type": "object",
+      "required": [
+        "ipAddress",
+        "endPointName",
+        "port",
+        "protocol"
+      ],
+      "properties": {
+        "ipAddress": {
+          "type": "string",
+          "description": "IP Address."
+        },
+        "endPointName": {
+          "type": "string",
+          "description": "Name of an end point."
+        },
+        "port": {
+          "type": "string",
```

Comment:
>type [](http://example.com/codeflow?start=11&length=4)

Just in case its an accidental authoring error that slipped by in previous versions. Do you really model it as string and not integer?

Reply:
yes, we model it as a string

Reply:
yeah we model it as string